pyquil/api/qpu.py

```diff
@@ -136,8 +153,8 @@ def run_and_measure(self, quil_program, qubits, trials=1):
         :return: A list of a list of bits.
         :rtype: list
         """
-        payload = self._run_and_measure_payload(quil_program, qubits, trials)
-
+        full_program = append_measures_to_program(quil_program, qubits)
+        payload = self._run_and_measure_payload(full_program, qubits, trials)
```

Comment:
This isn't using something like `_run_payload` because this bridge PR is one step behind an "expose run" PR?

Reply:
yes, the functionality will be merged in the next PR